### DISCUSSION TITLE
Saleor Updates which files?

### DISCUSSION THREAD

**Discussion by palash-cj** [saleor/saleor#12275] [General]
When saleor upgrades itself does it update only the 'core' named directory or anywhere within the saleor directory?

#saleor #upgrade #saleor-platform #saleor-dashboard #core

**Comment by timuric** [comment]
Hey @palash-cj any file in the repository can be changed

**Reply by palash-cj** [reply]
@timuric Thanks for the quick reply. 
This means if I create plugin and upgrade saleor, my plugin will get vanish. Then again I have to add plugin to the latest version code manually. Is this true? 

If so how can I add more functionalities I need for my business in the saleor?
